src/_data/starters/11ty-hbs-webpack.json

```diff
@@ -0,0 +1,7 @@
+{
+	"url": "https://github.com/uuki/11ty-hbs-webpack",
+	"name": "11ty-hbs-webpack",
+	"description": "Eleventy with Handlebars(support asynchronous helper) and Webpack boilerplate.",
+	"author": "uuki",
```

Comment:
I believe this needs to be a Twitter username for the avatar pic

Reply:
Thank you for your confirmation! I updated the value with this commit. 8af55f9